src/main/resources/locales/es-es.yml

```diff
@@ -25,9 +25,20 @@ edit_name_title: 'Editar Nombre of %1%'
 edit_description_title: 'Editar Descripcion of %1%'
 edit_privacy_button: '[Editar Privacidad](#00fb9a)'
 delete_button: '[Eliminar](#00fb9a)'
+delete_button_describe: '[  • Click Derecho: Eliminar](gray)'
```

Comment:
Is this google translate? If not, that's fine -- but otherwise, best just copy the english locales and let international contributors work on this